astropy/units/tests/test_typestubs.py

```diff
@@ -1,61 +0,0 @@
-import textwrap
```

Comment:
Am not sure the test file should be removed at all, but if it is, it should be done in the PR that introduces a new CI entry, not here. 

p.s. Obviously, it is fine to change what exactly is being checked.

Reply:
Before 487eec38ba3ca804fd5a81d334eb328da79bef23 the stub file for CDS units contained syntax errors, but these tests didn't catch that. Therefore they are completely useless.